web/src/components/ProductListPage/ProductListFilterSummary/ProductListFilterSummary.tsx

```diff
@@ -73,7 +73,7 @@ export const ProductListFilterSummary: FC<ProductListFilterSummaryProps> = ({
 			sorting={sorting}
 		>
 			{resultCount === 0 ? (
-				"No results found"
+				<></>
 			) : (
```

Comment:
Could we tidy up the condition inside this component to this instead?

`<>{resultCount > 0 && `Showing ${firstResult} to ${lastResult} of ${resultCount}`}</>`